src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -0,0 +1,185 @@
+  --pf-c-context-selector__toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-context-selector__toggle--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-context-selector__toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-context-selector__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-context-selector__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+
+  // Toggle Text
+  --pf-c-context-selector__toggle-text--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-context-selector__toggle-text--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-context-selector__toggle-text--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-context-selector__toggle-text--MaxWidth: calc(var(--pf-c-context-selector--Width) - var(--pf-c-context-selector__toggle-icon--MarginLeft) - 30px); // hard coded last variable
+
+  // Toggle Icon Arrow
+  --pf-c-context-selector__toggle-icon--MarginLeft: var(--pf-global--spacer--md);
+
+  // Menu
+  --pf-c-context-selector__menu--Top: calc(100% + var(--pf-global--spacer--xs));
+  --pf-c-context-selector__menu--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-context-selector__menu--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__menu--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-context-selector__menu--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-context-selector__menu--BoxShadow: var(--pf-global--BoxShadow--md);
+
+  // Menu Input
+  --pf-c-context-selector__menu-input--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__menu-input--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-input--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-input--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-input--BottomBorderColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-context-selector__menu-input--BottomBorderWidth: var(--pf-global--BorderWidth--sm);
+
+  // Menu Menu Item
+  --pf-c-context-selector__menu-list--MaxHeight: #{pf-size-prem(200px)};
+
+  // Menu Item
+  --pf-c-context-selector__menu-list-item--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__menu-list-item--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-context-selector__menu-list-item--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__menu-list-item--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-context-selector__menu-list-item--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-context-selector__menu-list-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-context-selector__menu-list-item--disabled--Color: var(--pf-global--Color--dark-200);
+  --pf-c-context-selector__menu-list-item--disabled--BackgroundColor: transparent;
+
+
```

Comment:
can you add `@extend %pf-t-light;` and remove the transparent backgrounds on lines 12 and 61. This will fix the theme issue.